src/components/donate/CampaignCard.tsx

```diff
@@ -30,69 +30,79 @@ const CampaignCard: FC<{
-              height: 150,
-              backgroundColor: "var(--blue-9)",
-            }}
-          />
-          <Flex
-            direction="row"
-            align="center"
-            gapX="3"
-            px="4"
-            py="3"
-            style={{
-              backgroundColor: "var(--green-9)",
-            }}
-          >
-            <Progress
-              value={raised}
-              variant="surface"
+        <Flex direction="column" justify={"between"} height={"100%"}>
+          <Inset clip="padding-box" pb="current">
+            <Image
+              src={imgSrc}
+              alt={imgAlt}
+              height={0}
+              width={0}
+              sizes={"100vw"}
               style={{
-                backgroundColor: "var(--green-4)",
+                display: "block",
+                objectFit: "cover",
+                width: "100%",
+                height: 150,
+                backgroundColor: "var(--blue-9)",
               }}
             />
-            <Text size="2" color="blue" highContrast>
-              {raised}%
-            </Text>
-          </Flex>
-        </Inset>
-        <Box mx="1" my="3">
-          <Heading as="h3" size="8" weight="regular" color="blue" highContrast>
-            {title}
-          </Heading>
-          <Box asChild mb="4">
-            <Text size="3">Goal: €{goal}</Text>
-          </Box>
-          <Flex gapX="2">
-            <Box px="4" py="2" asChild>
-              <Button asChild>
-                <Link href={donateLink} size="3">
-                  Donate now
-                </Link>
-              </Button>
-            </Box>
-            <Box px="4" py="2" asChild>
-              <Button variant="soft" asChild>
-                <Link href={moreLink} size="3" className="no-underline">
-                  More info
-                </Link>
-              </Button>
+            <Flex
+              direction="row"
+              align="center"
+              gapX="3"
+              px="4"
+              py="3"
+              style={{
+                backgroundColor: "var(--green-9)",
+              }}
+            >
+              <Progress
+                value={raised}
+                variant="surface"
+                style={{
+                  backgroundColor: "var(--green-4)",
+                }}
+              />
+              <Text size="2" color="blue" highContrast>
+                {raised}%
+              </Text>
+            </Flex>
+          </Inset>
+          <Box mx="1" mt="3">
+            <Heading
+              as="h3"
+              size="8"
+              weight="regular"
+              color="blue"
+              highContrast
+            >
+              {title}
+            </Heading>
+            <Box asChild mb="4">
+              <Text size="3">Goal: €{goal}</Text>
             </Box>
-          </Flex>
-        </Box>
+          </Box>
+          <Box mx="1" mb="3" mt="auto">
+            <Flex gap="2" wrap={"wrap"}>
```

Comment:
**suggestion (blocking):** The `Flex` component also has the standard margin / padding / positioning attributes available on layout components in Radix!  So instead of wrapping it in a `Box` you can just set those margin properties directly.